ci/script.sh

```diff
@@ -7,7 +7,7 @@ main() {
     cross build --target $TARGET
     cross build --target $TARGET --release
 
-    if [ -n $DISABLE_TESTS ]; then
+    if [ ! -z $DISABLE_TESTS ]; then
```

Comment:
Quotes would fix it. `-n` is used in v0.1.1 of `trust` while the `-z` version is used on the HEAD of the master branch.  I went with `-z` figuring it'll be switched to that anyway at some point.